Onboarding/Sources/Onboarding/ViewModels/OnboardingContainerViewModel.swift

```diff
@@ -0,0 +1,90 @@
+import Foundation
+import SwiftUI
+
+class OnboardingContainerViewModel: ObservableObject {
+    @Published var tabIndex: Int = 0
+    @Published var state = State.loading
+    @Published var slideCount: Int = 0
+    let skipButtonTitle = NSLocalizedString(
+        "skipButtonTitle",
+        bundle: .module,
+        comment: ""
+    )
```

Comment:
In `GDSCommon` we have an overload of `NSLocalizedString` function that makes the API a bit nicer for how we end up using it. If we really don't want to import `GDSCommon` perhaps we should re-implement the functionality?

Reply:
ive now created this extension for localisation
extension String {
  func localized() -> String {
    return NSLocalizedString(self, bundle: .module, comment: self)
  }
}

Reply:
I wonder if it would be better to be an overload of `NSLocalizedString` so that we keep some consistency with the apple SDK and in time, move it to a shared library - there doesn't seem much point in replicating this in each module. 

However, in a similar fashion, I don't think we would want to have different implementations for localising strings in each module either?

Reply:
done ive wrapped NSLocalised strings